.github/issue_template.md

```diff
@@ -1,21 +1,21 @@
-<!-- Thanks for the interest in Spring Cloud Data Flow! Before you create a new issue, please answer the following question.-->
+<!-- Thanks for your interest in Spring Cloud Data Flow! Before you create a new issue, please answer the following question.-->
 
-**Is this a "Bug Report" or "Feature Request"?** (choose one and remove the irrelevant sections before submitting the issue):
+**Is this a "Bug Report" or "Feature Request"?** (choose one and remove the irrelevant sections before you submit the issue):
 
 <!--
-If this is a "Bug Report", please:
+If this is a "Bug Report", please provide the following information:
 
-Describe:
-A clear and concise description of what the bug is. Additionally, it'd help if you can include the logs and entire stacktrace including the "caused by". (see GitHub-Markdown docs at: https://guides.github.com/features/mastering-markdown for logs/code formatting guidelines)_
+Description:
+A clear and concise description of what the bug is. Additionally, it would help if you could include the logs and the entire stacktrace, including the "caused by" portion. (see GitHub-Markdown docs at: https://guides.github.com/features/mastering-markdown for logs/code formatting guidelines)_
```

Comment:
I forgot to remove `_` towards the end. Please, if you can nuke that, that'd be great.